### DISCUSSION TITLE
Persian Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#358] [Language Leaders]
@AwesomeGuy000 - are you interested in being the Persian language leader?

**Comment by 0x414c49** [comment]
@synesthesiam I can be the Persian language leader.

**Reply by synesthesiam** [reply]
Great, thanks @0x414c49! Have you submitted any PR's yet for Persian? We ask that language leaders submit two small PR's to demonstrate knowledge of Github.

**Reply by 0x414c49** [reply]
@synesthesiam not yet, I was just asking to avoid overlaps.  I'm a software engineer and familiar with GitHub. I'll make a PR soon.

**Reply by synesthesiam** [reply]
Perfect, thank you. There can be multiple language leaders, but we don't currently have anyone for Persian.
